jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -16,6 +16,10 @@
 
 package org.jbpm.workbench.common.client.list;
 
+import static org.jbpm.workbench.common.client.PerspectiveIds.SEARCH_PARAMETER_IS_ERROR_ACK;
+import static org.jbpm.workbench.common.client.PerspectiveIds.SEARCH_PARAMETER_PROCESS_INSTANCE_ID;
+import static org.kie.workbench.common.workbench.client.PerspectiveIds.EXECUTION_ERRORS;
```

Comment:
perhaps also use _org.jbpm.workbench.common.client.PerspectiveIds_ in here to keep consistency.